lib/uplink/ext/uplink-cgo.h

```diff
@@ -0,0 +1,105 @@
+/* Code generated by cmd/cgo; DO NOT EDIT. */
```

Comment:
I don't think this should be checked in, it seems like a by production of compilation.

Reply:
My thought was that it was very similar to the sort of thing that is produced by dbx and/or protobuf and we do commit that stuff.

Reply:
This is more similar to an `.exe` rather than anything else. I.e. nothing in our Go codebase should directly depend on it, AFAIU.